docs/pages/configuration/pipelines/functions/build_images.mdx

```diff
@@ -5,37 +5,55 @@ sidebar_label: build_images
 
 The function `build_images` builds either all images (`--all` flag) or the images provided as arguments (e.g. `build_images api ui`). Images are, by default, processed in parallel.
 
+## Arguments
 
-## Args
+### `--image`
 
+The image you want to have built.
+
+### Usage
+
+```
+build_image image1 image2
+
+```
 
 ## Flags
 
+By default, flags are disabled. If any of the flags below are enabled, the following actions are taken.
+
 ### `--all`
 
-### `--tag / -t`
+All images will be built.
 
-### `--force-rebuild`
+### `--tag / -t`
 
-### `--max-concurrent`
+The build image tag will be returned.
 
-### `--sequential`
+### `--skip-build`
 
-### `--skip`
+The image will not be built.
 
 ### `--skip-push`
 
+The image will not be pushed to a container.
+
 ### `--skip-push-on-local-kubernetes`
 
+The image will not be pushed to a local kubernetes cluster.
 
-## Config Overwrite Flags
+### `--force-rebuild`
 
-\#TODO: This needs to be a reusable section
+The images will be rebuilt.
```

Comment:
Maybe use the word force here as image building is skipped if DevSpace determines that a build is not necessary

Reply:
fixed